src/upload.js

```diff
@@ -51,3 +51,14 @@ export async function uploadDirectory(portalUrl, directory, filename, options =
 
   return data;
 }
+
+/**
+ * Sometimes file object might have had the type property defined menually with Object.defineProperty
+ * and some browsers (namely firefox) can have problems reading it after the file has been appended
+ * to form data. To overcome this, we recreate the file object using native File constructor with
+ * a type defined as a constructor argument.
+ * Related issue: https://github.com/NebulousLabs/skynet-webportal/issues/290
+ */
+export function ensureFileObjectConsistency(file) {
```

Comment:
Is there any need to export the function?

Reply:
```suggestion
function ensureFileObjectConsistency(file) {
```